source/Organism.h

```diff
@@ -71,7 +71,7 @@ class Organism {
   virtual void VerticalTransmission(emp::Ptr<Organism> host_baby) {
     std::cout << "reproduce called from Organism" << std::endl;
     throw "Organism method called!";}
-  virtual double ProcessResources(double sym_piece){
```

Comment:
If Symbiont still has a ProcessResources with only one parameter, we need this here still